### DISCUSSION TITLE
Plesk docker

### DISCUSSION THREAD

**Discussion by Rar9** [directus/directus#15565] [Archive]
### Describe the Bug

When I save a directus docker instance on a ubuntu 22.04 plesk environment where host:container have been defined

Like
 /var/www/vhosts/domain/subdomain/uploads:/directus/uploads 

same for extensions &  databases

Empty volumns are added when saving 

In addition Mariadb is defind,  but again a DB_FILENAME:directus/database/database.sqlite

Is added 

This just doesn't make any sense. 

![Screenshot_20220913-151153_Plesk Mobile](https://user-images.githubusercontent.com/2239608/190075404-c43bb031-80f9-4bee-ade3-f1ddaac40830.jpg)

### To Reproduce

Used plesk with docker

### Errors Shown

See screenshot 

### What version of Directus are you using?

Latest

### What version of Node.js are you using?

16.7

### What database are you using?

Mariadb 10.x

### What browser are you using?

Firefox 

### How are you deploying Directus?

Docker on plesk

**Comment by azrikahar** [comment]
> Like
/var/www/vhosts/domain/subdomain/uploads:/directus/uploads

Huge disclaimer in that I don't personally have experience with Plesk, but based on what I'm seeing from their docs for Docker over here: https://docs.plesk.com/en-US/obsidian/administrator-guide/plesk-administration/using-docker.75823/

It is saying the _opposite_ of what we usually do with docker-compose:

![image](https://user-images.githubusercontent.com/42867097/190077391-8825bc0f-d222-4b7d-96ec-62c7de871e15.png)

So perhaps you should flip them in the Plesk UI? (which once again I'm not familiar with, so if you can provide screenshots on your configurations via Plesk, it'd be super helpful!)

> but again a DB_FILENAME:directus/database/database.sqlite

This is likely just the default `DB_FILENAME` environment variable set within the Dockerfile here: https://github.com/directus/directus/blob/9b24233aa4770024bb7a853142e596d00c3fe654/docker/Dockerfile#L31

Assuming you have changed `DB_CLIENT` to `mysql`, `DB_FILENAME` will _not_ be used at all, so it shouldn't affect anything.

**Reply by Rar9** [reply]
Ping me via discord, so this install can be resolved in ca ll.

**Reply by azrikahar** [reply]
Can you share what have you tried so far? Is the Plesk docs about volume mapping not working in your setup?

I think it's best to keep this conversation going over here so that other Plesk users can benefit from and/or share their Plesk docker setup in this discussion as well 🙂

**Comment by Rar9** [comment]
yes it seams the host/controller where switch ... my fault

Ubuntu redis is linked to docker container, but 

Now i´m getting  [cache] Error: connect ETIMEDOUT

![image](https://user-images.githubusercontent.com/2239608/190162136-dfec87b3-8362-46cc-9082-e4ecb0d524f7.png)

**Reply by azrikahar** [reply]
> Ubuntu redis is linked to docker container

Can you share more detail on the relevant configurations for this? Based on the error, I assume the cache and/or redis config is not 100% correct.
